test/flint/fmpz_factor-test.jl

```diff
@@ -0,0 +1,59 @@
+
+@testset "Integer factorization" begin
+
+  @test_throws ArgumentError factor(0)
+  @test_throws ArgumentError factor(Int128(0))
+  @test_throws ArgumentError factor(UInt128(0))
+  @test_throws ArgumentError factor(BigInt(0))
+  @test_throws ArgumentError factor(ZZ(0))
+
+
+  @test_throws DomainError factor(true)
+  @test_throws DomainError factor(false)
+
+
+  # trivial case: input is 1
+  F1 = factor(1)
+  F1_ZZ = factor(ZZ(1))
+  @test length(F1.fac) == 0
```

Comment:
```suggestion
  @test length(F1) == 0
```
this should work

Reply:
OK, thanks for the suggested clean up.  I have just checked in the relevant changes.

Reply:
@JohnAAbbott Maybe you checked them in but did not push them? They don't appear to be in the merged code. Perhaps send a follow-up PR to clean that up?